client/data/talks.json

```diff
@@ -449,6 +449,105 @@
       "video_url": "https://www.youtube.com/watch?v=MBndZddVQdw",
       "video_upload_date": "2019-06-03",
       "conferences": [1]
+    },
+    "50": {
```

Comment:
To fix the merge conflicts, merge the original master branch and update your IDs. Please re-order them as well so the numbers are in a logical order.